presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentDefaults.java

```diff
@@ -13,8 +13,6 @@
  */
 package io.prestosql.tests.product.launcher.env;
 
-import static java.lang.System.getenv;
-
 public class EnvironmentDefaults
 {
     public static final String DOCKER_IMAGES_VERSION = "31";
```

Comment:
why do we need those getters after this change? Can't we just use constants?

Reply:
Good point. Removing.